### ISSUE TITLE
Bug: Error in /openim/openim-server/scripts/start-all.sh:64   exit: Script openim-api.sh is missing or not executable.: numeric argument required

### ISSUE THREAD

**Issue by lgdzz** [openimsdk/open-im-server#1495]
## Starting OpenIM services
!!! [1129 15:51:34] Call tree:
!!! [1129 15:51:34]  1: /openim/openim-server/scripts/start-all.sh:78 execute_scripts(...)
!!! Error in /openim/openim-server/scripts/start-all.sh:64 
  Error in /openim/openim-server/scripts/start-all.sh:64. 'openim::log::error_exit "Error in ${BASH_SOURCE[1]}:${BASH_LINENO[0]}. '${BASH_COMMAND}' exited with status ${err}" "${1:-1}" 1' exited with status 1
Call stack:
  1: /openim/openim-server/scripts/start-all.sh:64 execute_scripts(...)
  2: /openim/openim-server/scripts/start-all.sh:78 main(...)
Exiting with status Script openim-api.sh is missing or not executable.
/openim/openim-server/scripts/lib/logging.sh: line 127: exit: Script openim-api.sh is missing or not executable.: numeric argument required
!!! Error in /openim/openim-server/scripts/docker-start-all.sh:31 
  Error in /openim/openim-server/scripts/docker-start-all.sh:31. '"${OPENIM_ROOT}"/scripts/start-all.sh' exited with status 2
Call stack:
  1: /openim/openim-server/scripts/docker-start-all.sh:31 main(...)
Exiting with status 1

### What did you expect to happen?

openim-server服务在运行10秒左右报错

### How can we reproduce it (as minimally and precisely as possible)?

按部署方式：
由于OpenIM涉及较多组件，部署麻烦，经过官方多次优化后，最新的部署文档如下：
docker 集成了web pc版 开箱即用，也增加了健康监测
git clone https://github.com/openim-sigs/openim-docker openim/openim-docker && export openim=$(pwd)/openim && cd $openim/openim-docker && make init && docker compose up -d
浏览器访问 http://ip:11001 即可使用pc web 聊天
具体参考文档：https://doc.rentsoft.cn/guides/gettingStarted/dockerCompose

### version

<details>
```console
$ {name} version
# paste output here
```
</details>

### Cloud provider

<details>
</details>

### OS version

<details>
```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here
# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```
</details>

### Install tools

<details>
</details>

**Comment by lgdzz** [comment]
Zookeeper starts successfully
Starting Kafka failed: kafka: client has run out of available brokers to talk to: read tcp 172.28.0.9:57814->172.28.0.1:19094: read: connection reset by peer
Checking components Round 5...
Mysql starts successfully
Mongo starts successfully
Minio starts successfully
2023/12/05 10:16:56 connected to 172.28.0.1:12181
Redis starts successfully
2023/12/05 10:16:56 authenticated: id=72091414166241292, timeout=4000
2023/12/05 10:16:56 re-submitting `0` credentials after reconnect
Zookeeper starts successfully
2023/12/05 10:16:56 recv loop terminated: EOF
2023/12/05 10:16:56 send loop terminated: <nil>
Kafka starts successfully
All components started successfully!

## Starting OpenIM services
!!! [1205 10:16:56] Call tree:
!!! [1205 10:16:56]  1: /openim/openim-server/scripts/start-all.sh:78 execute_scripts(...)
!!! Error in /openim/openim-server/scripts/start-all.sh:64 
  Error in /openim/openim-server/scripts/start-all.sh:64. 'openim::log::error_exit "Error in ${BASH_SOURCE[1]}:${BASH_LINENO[0]}. '${BASH_COMMAND}' exited with status ${err}" "${1:-1}" 1' exited with status 1
Call stack:
  1: /openim/openim-server/scripts/start-all.sh:64 execute_scripts(...)
  2: /openim/openim-server/scripts/start-all.sh:78 main(...)
Exiting with status Script openim-api.sh is missing or not executable.
/openim/openim-server/scripts/lib/logging.sh: line 127: exit: Script openim-api.sh is missing or not executable.: numeric argument required
!!! Error in /openim/openim-server/scripts/docker-start-all.sh:31 
  Error in /openim/openim-server/scripts/docker-start-all.sh:31. '"${OPENIM_ROOT}"/scripts/start-all.sh' exited with status 2
Call stack:
  1: /openim/openim-server/scripts/docker-start-all.sh:31 main(...)
Exiting with status 1





[root@VM-16-16-centos data]# echo $BASH_VERSION
4.2.46(2)-release

**Comment by lgdzz** [comment]
改用源码部署问题解决。
